### PR TITLE
TiDB Cloud Integration: Add metrics and update the dashboard

### PR DESCRIPTION
**Pull request by zywzoe**
### What does this PR do?

We add new metrics of tidb_cloud integration to improve user experience.

Two things were changed:

- Metrics in metadata.csv
- Dashboard

### Motivation

To improve our customers' experience in using TiDB Cloud with Datadog platform.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### CODE REVIEW

**Comment by lilyjazz** [comment]
Hi @fanny-jiang.
Cloud you please take some time to review this pr?
THX!

File: tidb_cloud/metadata.csv
Comment (lilyjazz) [inline tidb_cloud/metadata.csv:15]:
```suggestion
tidb_cloud.db_average_query_duration,gauge,,second,,"The duration between the time that the client's network request is sent to TiDB and the time that the request is returned to the client after TiDB has executed it.",0,tidb_cloud,,
```

File: tidb_cloud/metadata.csv
Comment (lilyjazz) [inline tidb_cloud/metadata.csv:14]:
```suggestion
tidb_cloud.db_queries_using_plan_cache_ops,gauge,,query,,"The statistics of queries using Plan Cache per second. The execution plan cache only supports the prepared statement command. If the value is 0, the execution plan cache has not been hit.",0,tidb_cloud,,
```

File: tidb_cloud/metadata.csv
Comment (lilyjazz) [inline tidb_cloud/metadata.csv:7]:
```suggestion
tidb_cloud.db_total_connection,gauge,30,connection,second,The number of current connection,0,tidb_cloud,,
```

File: tidb_cloud/metadata.csv
Comment (lilyjazz) [inline tidb_cloud/metadata.csv:8]:
```suggestion
tidb_cloud.db_active_connections,gauge,30,connection,second,The number of active connection,0,tidb_cloud,,
```

File: tidb_cloud/metadata.csv
Comment (lilyjazz) [inline tidb_cloud/metadata.csv:9]:
```suggestion
tidb_cloud.db_disconnections,gauge,30,connection,second,The number of clients disconnected,0,tidb_cloud,,
```

File: tidb_cloud/metadata.csv
Comment (lilyjazz) [inline tidb_cloud/metadata.csv:10]:
```suggestion
tidb_cloud.db_database_time,gauge,30,second,,A time model statistic that is the sum of all process' CPU consumption plus the sum of non-idle wait time.,0,tidb_cloud,,
```

File: tidb_cloud/metadata.csv
Comment (lilyjazz) [inline tidb_cloud/metadata.csv:11]:
```suggestion
tidb_cloud.db_query_per_second,gauge,30,query,second,"The number of SQL statements executed per second on all TiDB instances, which is counted according to SELECT, INSERT, UPDATE, and other types of statements",0,tidb_cloud,,
```

File: tidb_cloud/metadata.csv
Comment (lilyjazz) [inline tidb_cloud/metadata.csv:12]:
```suggestion
tidb_cloud.db_failed_queries,gauge,30,query,second,"The statistics of error types (such as syntax errors and primary key conflicts) according to the errors occurred when executing SQL statements per second on each TiDB instance.",0,tidb_cloud,,
```

File: tidb_cloud/metadata.csv
Comment (lilyjazz) [inline tidb_cloud/metadata.csv:13]:
```suggestion
tidb_cloud.db_command_per_second,gauge,30,command,second,"The number of commands processed by TiDB per second, which is classified according to the success or failure of command execution results",0,tidb_cloud,,
```

File: tidb_cloud/metadata.csv
Comment (lilyjazz) [inline tidb_cloud/metadata.csv:16]:
```suggestion
tidb_cloud.db_transaction_per_second,gauge,30,transaction,second,The number of transactions executed per second,0,tidb_cloud,,
```

**Review by lilyjazz — APPROVED** [review]

**Review by jose-manuel-almaza — APPROVED** [review]

**Comment by lilyjazz** [comment]
hi, @jose-manuel-almaza , I would like to know when this pr will merge?

**Comment by jose-manuel-almaza** [comment]
@zywzoe has the approval, waiting for him/her to merge

**Comment by lilyjazz** [comment]
@jose-manuel-almaza 
Thanks for merging this pr, but I found the change has not been implemented on the DATADOG UI.
Is there anything else we need to do to make this change?

**Comment by lilyjazz** [comment]
@jose-manuel-almaza 
Do we need to change manifest.json to trigger this change? such as, manifest_version?

**Pull request merged**
